### PR TITLE
Fix tooManyRedirects error

### PR DESCRIPTION
**Pull request by pushshift**
This pull request addresses issue: https://github.com/mitmedialab/feed_seeker/issues/4

### CODE REVIEW

File: feed_seeker/feed_seeker.py [hunk 1]
```diff
@@ -108,7 +108,7 @@ def default_fetch_function(url):
             return ''
 
     # ConnectionError for 404s, InvalidSchema for email addresses
```
Comment (ColCarroll) [inline feed_seeker/feed_seeker.py]: looks good -- would you mind updating the comment just above too?

Comment (pushshift) [reply feed_seeker/feed_seeker.py]: Sure thing.

**Comment by ColCarroll** [comment]
Oops, looks like my access to this repo and the travis build was taken away.

This looks good to me, thanks for the contribution @pushshift!

**Comment by hroberts** [comment]
looks good.  thanks for the review, @ColCarroll.

**Pull request merged**